probe_scraper/parsers/events.py

```diff
@@ -26,7 +26,7 @@ def extract_events_data(e):
         "expiry_version": "never",
         "expiry_day": "never",
         "name": e.methods[0],
-        "description": e.description,
+        "description": e._definition.get("description"),
```

Comment:
vendored parser no longer exposes `description`, I'm uncertain whether this line should instead be deleted.